pkg/volume/csi/expander.go

```diff
@@ -62,5 +69,28 @@ func (c *csiPlugin) NodeExpand(spec *volume.Spec, devicePath, deviceMountPath st
 	if err != nil {
 		return false, fmt.Errorf("Expander.NodeExpand failed to check if node supports expansion : %v", err)
 	}
-	return false, nil
+
+	if !nodeExpandSet {
+		return false, fmt.Errorf("Expander.NodeExpand found CSI plugin %s to not support node expansion", c.GetPluginName())
```

Comment:
Does this show up as an event?  If so, can we make the message more user-friendly?